src/vm/jithelpers.cpp

```diff
@@ -613,13 +613,36 @@ HCIMPL1_V(UINT64, JIT_Dbl2ULng, double val)
 
     const double two63  = 2147483648.0 * 4294967296.0;
     UINT64 ret;
+#ifdef _TARGET_XARCH_
```

Comment:
@jkotas Could you recommend proper flag? We need to keep using upper one on Windows everywhere, and Linux for x86/x64.